buildpack/telemetry/metrics.py

```diff
@@ -176,25 +176,41 @@ def get_influx_registry():
 
 
 def get_statsd_registry():
+    allowed_metrics = os.getenv("APM_METRICS_FILTER_ALLOW")
+    denied_metrics = os.getenv("APM_METRICS_FILTER_DENY")
+
+    if allowed_metrics and not denied_metrics:
```

Comment:
```suggestion
    if allowed_metrics and denied_metrics is not None:
```
More of a nitpick. Do we need to support falsey prefixes like `0` or `false`

Reply:
Here we're looking for (denied_metrics is None), but yeah for being more explicit, I'm refactoring my version to this. 

I think we don't need to validate the content here because it can easily be understood by users and fixed even if they do some mistakes on the first attempts